### ISSUE TITLE
Add standalone attachment or note to a regular item

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#243]
Not sure what the UI for this would be. Drag and drop could work, but that'd be a bit hard if the items weren't close to one another. An "Add to Item" option in the context menu, and that pops up some sort of filterable items list in a popup? That sounds like a lot of effort, but a widget like that would probably come in handy for other things later. Could do drop-and-drop for now.

**Comment by michalrentka** [comment]
@dstillman we already have drag & drop supported here. I could implement item picker if needed.

**Comment by dstillman** [comment]
Oh, if this existed at the time, I may have been confused by the lack of highlight feedback when hovering over an item. Possible to add that? Example would be when dragging a reminder on top of another reminder in Reminders.

Also, currently no way to move a child attachment or note out of an item, right?

**Comment by michalrentka** [comment]
@dstillman unfortunately there seems to be a bug in iOS. When a cell has custom background color (which we do), then when you're dragging anything over it it doesn't show the selected background. I can force select the cell manually, but this doesn't work when editing, where we have selection of multiple cells. In that case you'd just select and deselect cells by dragging anything over them. I guess I could force it and we can disable drag & drop while editing?

**Comment by dstillman** [comment]
By editing you mean Select mode? We wouldn't want to disable drag-and-drop for that — that's what lets you drag multiple items into a collection.

**Comment by michalrentka** [comment]
@dstillman yes, select mode. Then I can change the intent to "copy" which shows a "+" on right side, but still won't select background or I can just use default background, which will look quite bad in selection mode. Unfortunately I don't see other options currently. Maybe this will be fixed in iOS 15.

**Comment by michalrentka** [comment]
@dstillman I added the "+" icon, the background selection would be best, but it's at least something.
